docs/schema/guidance/record-ids.rst

```diff
@@ -4,4 +4,53 @@
 Record identifiers
 ==================
 
-[Stub page]
+To publish beneficial ownership data in BODS format, an information management system MUST maintain record identifiers for each element (person, entity and relationship) in each beneficial ownership network.
+
+Record identifiers MUST be unique within the publisher's system and never reused to refer to a different element. 
+
+There are situations where a publisher's system can allow information about the same person, entity or relationship to be collected in relation to the beneficial ownership networks of two (or more) different subjects. In these cases, information from different sources about the same person, entity or relationship can exist side-by-side and needs to be represented in published data. In these cases, each set of information MUST have its own record identifier. (See the example below.)
+
+The value of an entity or person record identifier MAY be a function of: the identity of the entity or natural person in question, plus the identity of the subject of the beneficial ownership network in which the entity or person appears. See :ref:`guidance-identifiers` for guidance on representing the identity of entities and natural persons.  
+
+The identity of a relationship is constituted by the identities of its subject and interested party. Therefore the value of a relationship record identifier MAY be a function of: the record identifiers of its subject and interested party, plus the identity of the subject of the beneficial ownership network in which the relationship appears.
+
+Record identifiers are used for :ref:`representing-bo` and for :ref:`guidance-updating-data`.
+
+Example scenario
+----------------
+
+Private companies in a given jurisdiction are obliged to disclose to the Company Registration Authority: their legal owners and any beneficial owners with direct interests in the company. 
+
+In the declaration from Alpha Corp., it discloses one natural person as a beneficial owner (and legal owner) and it also discloses Beta Systems as a legal owner.
```

Comment:
There's a mix of passive/active voice here. 

Maybe 'Apha Corp. declares a natural person and Beta systems as its legal owners' or 'Alpha Corp. makes a declaration disclosing...' (also depends on how important distinction of legal and beneficial owners is)

Reply:
Yes - will edit.